.github/workflows/upload-tutorials-stats.yml

```
@@ -0,0 +1,33 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Page age
+
+# Controls when the workflow will run
+on:
+  schedule:
+  # Run this once per day.
```

Leave this line in temporarily to automatically run your workflow every time you push this change

But make sure to remove it before merging the PR!

```suggestion
on:
  pull # DO NOT MERGE
  schedule:
  # Run this once per day.
```